product_multiple_barcodes/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     'name': 'Product Multiple Barcodes',
-    "version": "13.0.1.0.0",
```

Comment:
If you are changing version of module, also update readme with version notes

Reply:
README.rst was added